server/tests/flight.test.js

```diff
@@ -0,0 +1,28 @@
+import request from 'supertest-as-promised';
```

Comment:
sounds alright to me. I was also looking at https://www.npmjs.com/package/drakov to mock endpoints using the api-blueprint document, but wont need it for the time being i suppose.